### PR TITLE
feat : shop tab added, landing poster added

### PR DESCRIPTION
**Pull request by mikesch803**
1. shop tab is added in navbar
2. style.css moved to css
3. login btn has background and color changed
4. landing poster is added

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -9,25 +9,29 @@
...
+    <link rel="stylesheet" href="../css/style.css">
     <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
 </head>
 
 <body>
     <div class="grid-layout">
         <header class="header">
             <div class="navbar">
-                <h1 class="navbar-title" ><a href="./product.html">Fit Store</a></h1>
+                <h1 class="navbar-title" ><a href="../index.html">Fit Store</a></h1>
```
Comment (arjun-computer-geek) [inline pages/login.html]: you can also use only / for index.html in href

Comment (mikesch803) [reply pages/login.html]: > you can also use only / for index.html in href

so i tried but not working

Comment (mikesch803) [reply pages/login.html]: > > you can also use only / for index.html in href
> 
> so i tried but not working

thanks, working now

**Review by arjun-computer-geek — APPROVED** [review]
Looks good. You can merge now

**Comment by shahbazibnafzal** [comment]
Landing page cover photo taking time to load, you can add lazy loading there.
Also your title of all the html pages is Login, you might consider changing it to your brand name or different for all the pages.

**Comment by mikesch803** [comment]
> Landing page cover photo taking time to load, you can add lazy loading there. Also your title of all the html pages is Login, you might consider changing it to your brand name or different for all the pages.

thanks, I made the changes

**Review by shahbazibnafzal — APPROVED** [review]
Looks good

**Pull request merged**